app/plot_app/configured_plots.py

```diff
@@ -339,6 +339,16 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data, link_to_3d_page,
                             ['Roll Rate Groundtruth', 'Pitch Rate Groundtruth',
                              'Yaw Rate Groundtruth'])
 
+        # Vision latency
+        data_plot = DataPlot(data, plot_config, 'vehicle_visual_odometry',
+                             y_axis_label='[ms]', title='Visual Odometry Latency',
+                             plot_height='small', changed_params=changed_params,
+                             x_range=x_range)
+        data_plot.add_graph([lambda data: ('latency', 1e-3*(data['timestamp'] - data['timestamp_sample']))
+                             ], colors3, ['VIO Latency'], mark_nan=True)
+        plot_flight_modes_background(
+            data_plot, flight_mode_changes, vtol_states)
+
         if data_plot.finalize() is not None: plots.append(data_plot)
```

Comment:
This line is missing for the previous plot now (`Vision attitude rate`)
Same for `Vision attitude`.